### PR TITLE
Firelock and Windoor Direction only

### PR DESCRIPTION
**Pull request by ThatDan123**
-Firelock and Windoor Direction only

-Cleaned up some of the rotation script

-Sprite will reset to initial variant not 0 on refresh if needed

### CODE REVIEW

File: UnityProject/Assets/Scripts/Objects/Directionals/Rotatable.cs [hunk 1]
```diff
@@ -261,7 +261,7 @@ public void ResitOthers()
 				}
 				else
 				{
-					spriteHandler.ChangeSpriteVariant(0, false);
+					spriteHandler.ChangeSpriteVariant(spriteHandler.InitialVariantIndex, false);
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/Objects/Directionals/Rotatable.cs]: initial variant index is too confusing

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Objects/Directionals/Rotatable.cs]: also I have changes here in the PR

Comment (ThatDan123) [reply UnityProject/Assets/Scripts/Objects/Directionals/Rotatable.cs]: but thats what the default should be??

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Objects/Directionals/Rotatable.cs]: well, it only runs it if it's not being set Rotational

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Objects/Directionals/Rotatable.cs]: if (ChangeSprites == false)

Comment (corp-0) [reply UnityProject/Assets/Scripts/Objects/Directionals/Rotatable.cs]: ChangeSprites sounds like a method, not a bool

Comment (ThatDan123) [reply UnityProject/Assets/Scripts/Objects/Directionals/Rotatable.cs]: Done, back to 0

**Review by corp-0 — APPROVED** [review]

**Pull request closed without merging**